src/Broadway/Saga/MultipleSagaManager.php

```diff
@@ -66,6 +95,14 @@ public function handle(DomainMessage $domainMessage)
                 // event with a new state.
                 $state = $this->stateManager->generateNewState();
                 $this->handleEventBySagaWithState($sagaType, $saga, $event, $state);
+            } elseif ($criteria instanceof CopiedCriteriaInterface) {
+                // For a copy a new state is needed with values from original event.
+                $states = $this->stateManager->findBy($criteria, $sagaType, false);
+                foreach ($states as $state) {
+                    // TODO: What if more then one state returned?
```

Comment:
Kunnen we hier expliciet een check op doen en tenminste iets loggen als dat zou voorvallen?
Misschien ook enkel een copy en handleEventBySagaWithState doen voor de eerste state die we vinden. Maar misschien niet noodzakelijk crashen...